MonkeyWrench/MonkeyWrench.csproj

```diff
@@ -62,7 +62,6 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AssemblyInfo.cs" />
```

Comment:
This was intentional because it broke my build otherwise. There was no AssemblyInfo file in the solution, I had to remove it in order to build at all. This _could_ be moved into a separate commit though, or was the file meant to be in the repo?